web-ui/src/main/resources/catalog/components/viewer/partials/mainviewer.html

```diff
@@ -40,6 +40,14 @@
       <span class="fa fa-print"></span>
       <span role="tooltip" data-translate="">Print</span>
     </button>
+
+    <button class="btn btn-default" type="submit"
+            data-ng-click="syncMod(map)">
+      <span class="fa fa-sync"
+            data-ng-class="synAllLayers ? 'fa-lock' : 'fa-unlock'"/>
+      <span role="tooltip" data-translate="">Sync All Layers</span>
```

Comment:
Usually, translation key does not contain spaces eg. filterData
and the key is missing in en-search.json (and if not there, it will not go to transifex).